### PR TITLE
Feature database

### PR DESCRIPTION
**Pull request by Mantas330BJ**
Implemented database with Post, Put, Delete, Get requests. Cart page branch is not included.

### CODE REVIEW

**Comment by artur-masalcev** [comment]
I don't know if it is a database problem, but I've found a bug related to adding a new deal to a food place. I recorded this clip that shows it more clearly (notice how the amount of deals changes when i reload the app)

https://user-images.githubusercontent.com/76208684/145576185-6a60dc6b-1754-4ddb-b5fb-63ab061b75f8.mp4

)

**Comment by artur-masalcev** [comment]
I guess it is time to remove the word "dummy" from all folders, classes, etc., related to data manipulation

**Comment by Mantas330BJ** [comment]
> I don't know if it is a database problem, but I've found a bug related to adding a new deal to a food place. I recorded this clip that shows it more clearly (notice how the amount of deals changes when I reload the app)
> 
>  wasted-feature-db-bug-1.mp4

Fixed

**Comment by Mantas330BJ** [comment]
> I guess it is time to remove the word "dummy" from all folders, classes, etc., related to data manipulation

Removed

**Review by artur-masalcev — APPROVED** [review]

File: DataAPI/Startup.cs [hunk 1]
```diff
@@ -15,9 +18,16 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddAutoMapper(typeof(Startup));
+            services.AddScoped<DealsRepository>(); //TODO: rename to singular
```
Comment (viliusjun) [inline DataAPI/Startup.cs:24]: Should we mark this as a "TODO" issue in Github?

**Review by viliusjun — APPROVED** [review]

File: DataAPI/Startup.cs (hunk 1 above)
Comment (Mantas330BJ) [reply DataAPI/Startup.cs:24]: That was just a silly comment that I forgot

File: DataAPI/Repositories/PlacesRepository.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            _dbContext = dbContext;
+        }
+
+        public IEnumerable<FoodPlace> Get()
+        {
+            return _dbContext.FoodPlaces
+                .Include(p => p.Deals)
+                .Include(p => p.Ratings)
+                .Include(p => p.PlaceType)
+                //.Include(p => p.PlaceUser)
```
[the remark this answers is not in the document]
Comment (Mantas330BJ) [reply DataAPI/Repositories/PlacesRepository.cs:22]: No

**Pull request merged**